index.js

```diff
@@ -6,13 +6,153 @@ function ask(questionText) {
+      else if (rangeInput === 'H') {
+        minNum = randNum + 1
+        randNum = Math.floor((maxNum + minNum) / 2)
+        return randNum;
+
+      }
+      //If the user isn't cheating and the user's number is lower than the random number, look at all values from the minimum, all the way to the (random number - 1) and find the middle of that range. Return that value.
+      else if (rangeInput === "L") {
+        maxNum = randNum - 1
+        randNum = Math.floor((maxNum + minNum) / 2)
+        return randNum;
+
+      }
+    }
+
+    //Loop that stops only when the user inputs "Y" for whether or not the computer guessed the number correctly.
+    while (userInput !== "Y") {
+      //Increment the guess count through each iteration of the while loop to see how many guesses the computer takes to get the number right
+      guessCount += 1
+      //Since we are already in this loop, it means the user inputted "N" when asked if the number was guessed correctly. If the user input N, but the guess is actually correct, the user is cheating.
+      if (parseInt(secretNumber) === randNum) {
+        console.log(`I can tell when you're cheating! Goodbye.  `)
+        process.exit()
+
+      }
+      
+      //Reassign the value of randNum to the return value of the rangeFunction. The value is different depending on whether the user says the number is higher or lower than the computer's randomly generated number.
+      randNum = await rangeFunction()
+      
+      //Ask the user again if the newly generated randNum is equal to the number the user has chosen.
+      userInput = await ask(`Is your number ${randNum}? Type 'Y' or 'N': `)
+
+      
+    }
+    //This is out of the while loop which means the user inputted "Y". If the user said yes but the two numbers are not equal, the user is lying.
+    if(parseInt(secretNumber) !== randNum){
+      console.log(`I can tell when you're cheating! Goodbye.  `)
+        process.exit()
+
+    }
+    //If the user inputs Y and is not cheating, print out that number along with the amount of tries it took to guess it.
+    console.log(`Your number is ${secretNumber}! I guessed it in ${guessCount} tries!`)
+    //Promp the user if they would like to play again, otherwise end the program. 
+    let playAgain = await ask("Would you like to play again? Type 'Y' or 'N': ")
+    if (playAgain === "Y") {
+      start()
+    }
+    else { 
+      console.log("Goodbye!")
+      
+      process.exit() 
+    
+    }
+
+
+  
+
+
 }
+//Function that initially asks the user to choose between the two games and depending on the input, runs the function that begins the selected game. 
+async function chooseGame(){
+  let userChoice = await ask("Would you rather play a game where you guess my number or a game where you guess mine? \nType '1' for the first option or '2' for the second: ")
```

Comment:
this console.log asks if we want to play the same kind of game twice, and is very misleading.

Reply:
This was just a complete typo on my part. I think my brain was kind of fried by that point so I ended up asking the same thing twice.